packages/manager/modules/pci/src/components/project/users/users.controller.js

```diff
@@ -4,14 +4,21 @@ import { ACTIVE_STATUS, PENDING_STATUS } from './users.constants';
 
 export default class CloudProjectUsersCtrl {
   /* @ngInject */
-  constructor($q, $translate, atInternet, CucCloudMessage) {
+  constructor($q, $translate, atInternet, CucCloudMessage, ovhFeatureFlipping) {
     this.$q = $q;
     this.$translate = $translate;
     this.atInternet = atInternet;
     this.CucCloudMessage = CucCloudMessage;
+    this.ovhFeatureFlipping = ovhFeatureFlipping;
   }
 
   $onInit() {
+    this.showGenerateS3Credentials = !this.ovhFeatureFlipping
+      .checkFeatureAvailability('public-cloud:object-storage')
+      .then((feature) =>
+        feature.isFeatureAvailable('public-cloud:object-storage'),
+      )
+      .then((status) => status);
```

Comment:
**issue(blocking)**: this will always evaluate to `false`
```suggestion
    this.ovhFeatureFlipping
      .checkFeatureAvailability('public-cloud:object-storage')
      .then((feature) =>
        feature.isFeatureAvailable('public-cloud:object-storage'),
      )
      .then((status) => {
      this.showGenerateS3Credentials = !status;
      });
```